Project.toml

```
@@ -23,7 +23,7 @@ ZygoteRules = "700de1a5-db45-46bc-99cf-38207098b444"
 
 [compat]
 AbstractFFTs = "0.5, 1.0"
-ChainRules = "1"
+ChainRules = "1.5"
 ChainRulesCore = "1.0.1"
```

Probably doesn't matter, but CR 1.5 needs CRC 1.1
```suggestion
ChainRulesCore = "1.1"
```